### PR TITLE
Docs/pam 176 rule editor

### PR DESCRIPTION
**Pull request by pamgoodrich**

### CODE REVIEW

**Comment by pamgoodrich** [comment]
@naumanali-stoplight : I think I've done enough for early access. We should have pretty good info for people to understand the feature. I highly recommend looking at it in meta:
https://meta.stoplight.io/docs/platform/branches/docs%2Fpam-176-rule-editor/ZG9jOjMwMDI3MjA2-overview

Note that I pulled the examples into their own topic and added some task navigation links to the bottom of each topics. 

It might be good to take a close look at the examples. 
https://meta.stoplight.io/docs/platform/branches/docs%2Fpam-176-rule-editor/ZG9jOjMwNDI1OTM1-rule-examples

I'll stay out of here until I get feedback from you.

Thank you!

File: 2a.-style-guides/f.refresh-style-guide.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Refresh Style Guides
```
Comment (mnaumanali94) [inline 2a.-style-guides/f.refresh-style-guide.md:1]: I'd use update than refresh across this article

File: 2a.-style-guides/c-create-rules.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
+# Create Rules
+
```
Comment (mnaumanali94) [inline 2a.-style-guides/c-create-rules.md:2]: This needs an early access callout

File: 2a.-style-guides/a.style-guide-projects.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+---
+tags: [Governance]
+---
+
+# Overview
+
+> Limited early access content for Shared Style Guides. Shared style guides are available on the **Professional** and **Enterprise** plans for early access participants.
```
Comment (mnaumanali94) [inline 2a.-style-guides/a.style-guide-projects.md:7]: Lets make these callouts warning rather than info.

**Review by mnaumanali94 — CHANGES_REQUESTED** [review]

**Comment by mnaumanali94** [comment]
Looks like a great first pass. Lets make these tiny changes and we can get this in.

**Review by mnaumanali94 — APPROVED** [review]

**Pull request merged**